src/WebJobs.Extensions/Extensions/ExtensionsWebJobsStartup.cs

```diff
@@ -13,7 +13,6 @@ public class ExtensionsWebJobsStartup : IWebJobsStartup
     {
         public void Configure(IWebJobsBuilder builder)
         {
-            builder.AddTimers();
```

Comment:
Timers is still part of the default extensions package, and should register its non-storage specific functionality. E.g. for someone running a TimerTrigger in a WebJob on a single instance, they won't have to pull in Timers.Storage and can use the file based implementation of ScheduleMonitor.


Previously to get storage based schedule tracking and distributed locking, a user had to call `builder.AddTimers()` and `builder.AddAzureStorageCoreServices()`. With your new Timers.Storage package, I'd expect you'd have a `builder.AddTimersStorage()` or something, and that guy could internally add the StorageScheduleMonitor, and could also call `builder.AddAzureStorageCoreServices()`.

Reply:
Added these changes. We have a new builder method in the Timers.Storage project for builder.AddTimersStorage() to add the blob storage backed schedule monitor. 

One other note is that FileSystemScheduleMonitor doesn't seem to work in the Host - DI defaults to the FileSystemScheduleMonitor(string currentDirectory) ctor instead of the parameterless ctor. Do we need a change here? The Host will use the builder.AddTimersStorage() but consumers of WebJobs.Extensions that choose to use File based schedule monitor may fail here.